test/gtest/uct/ib/test_rc.cc

```diff
@@ -806,7 +806,7 @@ test_uct_iface_attrs::attr_map_t test_rc_iface_attrs::get_num_iov() {
         EXPECT_TRUE(has_transport("rc_verbs"));
         m_e->connect(0, *m_e, 0);
         uct_rc_verbs_ep_t *ep = ucs_derived_of(m_e->ep(0), uct_rc_verbs_ep_t);
-        uint32_t max_sge;
+        uint32_t max_sge = 0; // for gcc 10 -Og
```

Comment:
Why?

Reply:
Without it getting error. Looks like compiler bug. 
```
test/gtest/uct/ib/test_rc.cc:814:34: error: 'max_sge' may be used uninitialized in this function [-Werror=maybe-uninitialized]
         iov_map["am"]  = max_sge - 1; // 1 iov reserved for am header
```